### ISSUE TITLE
Add option to choose plugins

### ISSUE THREAD

**Issue by benkeen** [benkeen/generatedata#302]
The plugin list has grown quite a bit and they're growing a little more specific. Time to add the option (maybe during the installation process, too) to allow users to select/unselect specific Data Types.

This may require adding a one-liner to each Data Type to describe it.

**Comment by NioZero** [comment]
it would be nice too to have an option to install plugins without refreshing everything, or at least select which plugin to refresh.

**Comment by benkeen** [comment]
I know what you mean, but actually it doesn't make any difference - well, other than taking a little longer. No information is lost when the plugins are reset and it guarantees that all newly added/removed plugins are accounted for.

**Comment by NioZero** [comment]
yes, is only for development purpose... I usually delete the others plugins until i finished...
